### ISSUE TITLE
CssSyntaxError: Unknown word for valid less

### ISSUE THREAD

**Issue by eddie-dunn** [stylelint/stylelint#3831]
> Clearly describe the bug

`stylelint` reports CssSyntaxError for ostensibly valid `.less`

> Which rule, if any, is the bug related to?

CssSyntaxError

> What CSS is needed to reproduce the bug?

```css
.test {
	.foo( {
		width: calc(~"100% + 1px");
	} );
}

```

> What stylelint configuration is needed to reproduce the bug?

```
module.exports = {
	'rules': {
		'comment-no-empty': true
	}
};
```

> Which version of stylelint are you using?

`9.9.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint "**/*.less" --config myconfig.js`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, LESS.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
web/test.less
 4:4  ✖  Unknown word   CssSyntaxError

error Command failed with exit code 2)
```

**Comment by evilebottnawi** [comment]
@eddie-dunn problem in `postcss-less`, better create issue in their repo

**Comment by eddie-dunn** [comment]
OK, thanks. Reported in https://github.com/shellscape/postcss-less/issues/128
